README.md

```diff
@@ -4,6 +4,29 @@ Open Search Scaling Manager is to scale up or scale down a node in a cluster bas
 
 
 
```

Comment:
Please add a description of what Scaling Manager is and what are the features build in it